### PR TITLE
Mac-Lakehead-CEM2594-Reach_Indicator

### PR DESCRIPTION
**Pull request by mfnisbetLU**
Initial Commit for CEM2594 Reach Indicator. To do: change colors to theme colors in progressbar, render progressbar and engagement box depending on if props have values (combine reach and engagement progress bars, and render the box depending on if post engagements and or people reached have values). Demo video below: 

https://user-images.githubusercontent.com/91333450/142470463-eb33740a-e3e8-4f37-829d-40851db89640.mp4

### CODE REVIEW

File: src/Containers/ReachIndicator/ReachIndicator.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import styled from 'styled-components';
+import { } from 'Themes';
+
+export interface ReachIndicatorProps 
+extends React.HTMLAttributes<HTMLDivElement> {
+    engageNum: number,
```
Comment (ralph-dev) [inline src/Containers/ReachIndicator/ReachIndicator.tsx:7]: Don't forget documentation I want to understand how these props are being used

File: src/Containers/ReachIndicator/ReachIndicator.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+= ({
+    engageNum,
+    engageGoal,
+    reachNum,
+    reachGoal,
+    percentMark,
+    ...props
+}): React.ReactElement => (
+    <ReachIndicatorFlex>
+        <EngagementBox {...props}>
```
Comment (ralph-dev) [inline src/Containers/ReachIndicator/ReachIndicator.tsx:24]: Remember you should have one engagement box and have two instances of the component next to each other

**Review by ralph-dev — COMMENTED** [review]
It's coming together!

**Review by ralph-dev — CHANGES_REQUESTED** [review]
Waiting for in meeting changes

**Comment by mfnisbetLU** [comment]
Sorry, I had the changes done this morning, but I wanted to make sure everything was right. I remember you asking for the display on a small mobile device, so attached is that, and a case for a very long indicator name.
Reach indicator small mobile
Reach indicator verbose

File: src/Containers/ReachIndicator/ReachIndicator.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export interface ReachIndicatorProps
+    extends React.HTMLAttributes<HTMLDivElement> {
+    indicatorName: string, // Reach indicator display name (people reached/post engagements)
```
Comment (ralph-dev) [inline src/Containers/ReachIndicator/ReachIndicator.tsx:6]: Format this correct 

/* reach */
indictora

File: src/Containers/ReachIndicator/ReachIndicator.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+}): React.ReactElement => (
+    <div>
+        { !!(indicatorNum > indicatorMinimum) && <>
+            <EngagementBox {...props}>
+                <EngagementHeader>{indicatorNum}</EngagementHeader>
+                <EngagementTheme>{indicatorName}</EngagementTheme>
+                <IndicatorProgressBar indicatorName={indicatorName} indicatorNum={indicatorNum} indicatorGoal={indicatorGoal} indicatorPercentMark={indicatorPercentMark} indicatorMinimum={indicatorMinimum} />
+                <LoadingBar />
+            </EngagementBox >
+            </>
```
Comment (ralph-dev) [inline src/Containers/ReachIndicator/ReachIndicator.tsx:29]: remove this empty one, you don't need it anymore

**Review by ralph-dev — APPROVED** [review]
LGTM, please fix small changes first

**Pull request merged**